### ISSUE TITLE
How to add horizontal scrollbar?

### ISSUE THREAD

**Issue by sk1e** [Diokuz/react-baron#13]
Does direction param 'h' stand for horizontal scrolling? 

With the following code:
```
  <div style={{ width: 500, height: 100 }}>
    <Baron params={{ direction: 'h' }}>
      <div style={{ width: 2000, height: 2000, background: 'green' }}>
        hello
      </div>
    </Baron>
  </div>
```
I don't get any scrollbar. With direction value as 'v' I can see a vertical scrollbar.

**Comment by Nitive** [comment]
The reason is `undefined` in track and bar classes.
After replacing it with `_h` everything works perfectly.

![image](https://user-images.githubusercontent.com/6770225/46068537-0777c880-c182-11e8-9ab6-436ed1c9974f.png)

**Comment by sk1e** [comment]
@Nitive oh, that's great. But component API seems to be broken.

P.S. Glad to see a guy from Tomsk here >_<

**Comment by Diokuz** [comment]
react-baron uses css-modules here https://github.com/Diokuz/react-baron/blob/master/src/component.js#L17
So, if it is undefined, probably something wrong with your webpack configuration.

**Comment by Nitive** [comment]
Looks like there is no default className for _h in es5 build:
https://github.com/Diokuz/react-baron/blob/master/webpack.config.js#L4-L8
but it has to be:
https://github.com/Diokuz/react-baron/blob/master/src/component.js#L19

**Comment by Diokuz** [comment]
Yes, just use `hModifier` prop to define it. https://github.com/Diokuz/react-baron/blob/master/src/component.js#L70-L71
